### ISSUE TITLE
Make highlight styles editable by user

### ISSUE THREAD

**Issue by nbalakin** [nbalakin/VSOutputEnhancer#2]
It should be possible to change text colors and decorations by user.

**Comment by sadovsf** [comment]
Also few custom (probably regex based) filters / colors would be awesome. Check out VSColorOutput for example ;-)

**Comment by MartinL0815** [comment]
Yes, please add the available style (foreground, background colors, bold text style) to the "fonts & colors" section of the VS2015/2017 options!
Or create you won settings within the options.
My background is, that I work in high-contrast-mode with a black window background and yellow window text.
I need to fine tune VS and the text coloring to be able to read everything and your extension does not allow this fine tuning!

**Comment by nbalakin** [comment]
Hi, @MartinL0815, sorry for long response.
As temporary workaround you can edit DarkTheme.json/LightTheme.json file in your VS extension folder.

**Comment by szalapski** [comment]
Even being sensitive to light mode would be nice.  I don't want to change the colors, I just want them a little darker when in light mode.
